theano/scalar/basic.py

```diff
@@ -1210,10 +1213,9 @@ def __hash__(self):
     def output_types(self, *input_dtypes):
         return [bool] if getattr(self, 'bool', False) else [int8]
 
-    def grad(self, inputs, output_gradients):
+    def L_op(self, inputs, out, output_gradients):
```

Comment:
All the PR seem good. I can merge it like this. But if you could make the small following change, it would be great. If you don't have more time, just tell me and I'll merge.

It would be great to be more consistent. I mean, here it use "inputs" and "out". Can you do a search and replace of "inputs, out," to "inputs, outputs," and "inputs, outs," to "inputs, outputs" and the needed adjustment in the methods? This way, the file will be more consistent and ease futur work in it. There isn,t many place that will be changed by those search and replace.

thanks